datus/utils/rerankers.py

```
@@ -26,6 +26,13 @@ def __init__(
         return_score: str = "relevance",
     ):
         """Initialize the BGE reranker."""
+        try:
+            import torch  # noqa: F401
```

Ruff (0.14.1)

Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)